torch_xla/csrc/data_ops.cpp

```diff
@@ -92,6 +92,17 @@ xla::XlaOp BuildView(xla::XlaOp input, absl::Span<const int64_t> output_sizes) {
   return XlaHelpers::DynamicReshape(input, complete_output_sizes);
 }
 
+xla::XlaOp SetDimensionSizes(xla::XlaOp input,
+                             absl::Span<const xla::XlaOp> output_sizes) {
+  for (int i = 0; i < output_sizes.size(); i++) {
```

Comment:
do we need to check whether input size and `output_sizes` has same length?

Reply:
Makes sense. Adding the protection.